src/cr/cube/cube.py

```diff
@@ -415,13 +415,19 @@ def _ca_as_0th(self):
     def _cube_dict(self):
         """dict containing raw cube response, parsed from JSON payload."""
         cube_dict = copy.deepcopy(self._cube_response)
+        if self._cube_idx_arg:
+            cube_dict["cube_idx"] = self._cube_idx_arg
         if self._mean_subvariables:
             dimensions = cube_dict.get("result", {}).get("dimensions", [])
             # ---cube inflation---
             # ---In case of numeric arrays, we need to inflate the columns dimension
             # ---according to the mean subvariables. For each subvar the col dimension
             # ---will have a new element related to the subvar metadata.
-            dimensions.append(self._numeric_array_dimension)
+            (
```

Comment:
This is not an assignment (nothing on the left side). I would therefore prefer the traditional structure (with `if/else`) instead of the ternary expression.